packages/plugins/users-permissions/admin/src/hooks/useForm/index.js

```diff
@@ -19,7 +18,7 @@ const useUserForm = (endPoint, permissions) => {
           type: 'GET_DATA',
         });
 
-        const data = await request(getRequestURL(endPoint), { method: 'GET', signal });
+        const { data } = await get(getRequestURL(endPoint));
```

Comment:
Same. Sure you can omit the signal?

Reply:
fetchClient has it's own `AbortController` it always passes 👍🏼